packages/ERTP/src/types.ts

```diff
@@ -405,16 +406,21 @@ export type PurseMethods<
 export type Payment<
   K extends AssetKind = AssetKind,
   M extends Key = Key,
-> = RemotableObject & PaymentMethods<K>;
-export type PaymentMethods<K extends AssetKind = AssetKind> = {
-  /**
-   * Get the allegedBrand, indicating
-   * the type of digital asset this payment purports to be, and which issuer to
-   * use. Because payments are not trusted, any method calls on payments should
-   * be treated with suspicion and verified elsewhere.
-   */
-  getAllegedBrand: () => Brand<K>;
-};
+> = RemotableObject &
+  Tagged<
```

Comment:
I understand that the unfortunate name collision on `Tagged` comes from elsewhere, so we cannot easily fix it. But especially when juxtaposed with `Remotable` or other passable type names, it is terribly confusing. Please at least comment all such usage sites.

I see you're passing this through `'@agoric/internal/src/tagged.js'` anyway. Perhaps we could have the reexport a rename, and then avoid this unfortunate name collision everywhere else? This would be better than commenting each confusing usage site.

Reply:
Fair. I've added an alternate export. I kept the rest of the source to make it easier to maintain with changes upstream